informus/rules/patients.md

```diff
@@ -1,65 +1,117 @@
+
+## Handling moved / discharged patients
+
+### T03 'wait' bed
+
+On T03 there is a 'wait' bed denoted by `T03^T03 WAITING^WAIT`. This 'wait' bed isn't used consistently, so it is unclear what it represents exactly. 
+
+All data from the 'wait' bed should be excluded from all informus data.
+
+### 'ghost' patients
+
+'ghost' patients appear as rows with a `NULL` `hospital_visit` admission datetime. All data from 'ghost' patients (i.e. hospital visits with a `NULL` admission) should be excluded from all informus data.
+
+### Tiles (i.e. `unit_wide` data)
+
+All tiles should include data from any patient that has been on the unit in the last 24 hours including: 
+
+- patients that have since been moved to another location in the hospital
+  
+- patients that have since been discharged from the hospital and/or died
+  
+The exceptions to this are: `bed occupancy` and `all targets set` - these tiles should only use data from patients currently present on the unit.
+
+For those that include discharged/moved patients, they should use all data produced by patients _on the current unit_ in the last 24 hours. For example, let's consider the tiles on the T03 dashboard. Let's imagine a patient has been on T03 for multiple days in a specific bed, before being moved 12 hours ago to another ward. In this scenario we would use all data generated by the patient on T03 in the last 24 hours (so from 12-24 hours ago), but ignore any data generated while they are off the unit (0-12 hours ago). No data generated outside of the unit should be included in any tile calculations.
+
+This also holds true for shorter trips in and out of the unit. For example, let's imagine a patient has been on T03 for multiple days in a specific bed, before being moved 12 hours ago to another location in the hospital (could be another ward, or the operating theatre, imaging etc.), before being transferred back to the same bed on T03 10 hours ago. In this scenario we would use all data generated by the patient on T03 in the last 24 hours (so 0-10 hours ago, as well as 12-24 hours ago), but ignore any data generated during the 2 hours they were off the unit.
+
+Equally, we must ensure that any data 'gaps' in the last 24 hours (due to the patient being outside the unit, in another location in the hospital), don't negatively impact the scores on each tile. There is no data for these patients on T03 in these gaps, so they shouldn't contribute to the overall score.
+
+Tiles should still show data if there are no patients currently on the unit (as long as there have been some moved / discharged patients present in the last 24 hours). Clicking on a tile should show an empty floorplan in this scenario.
+
+If there are no current patients and there haven't been any in the last 24 hours, then tiles should display text that says 'no data available'.
```

Comment:
A much more straightforward solution would be to show a message that says something like:

"There have been no patients on this unit in the last 24 hours". 

Considering the rarity of this scenario, I'd be tempted to implement this using the least logic we can.

Reply:
Makes sense to me! Would this mean the tiles don't show up in that scenario? Just the message?

Reply:
Yes, just a message with no Tiles shown.